setup.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
```

Comment:
You can basically ignore this file, since it's mostly just Python/Sideboard boilerplate.